app.js

```diff
@@ -1,14 +1,34 @@
+'use strict';
 
+/* External modules */
 
-var express = require('express'),
-  config = require('./config/config'),
-  glob = require('glob');
+var fs = require('fs');
+var express = require('express');
+var winston = require('winston');
+
+/* Library modules */
+
+var CampaignInfoStorage = require('./lib/storage/sqlite/campaign-info-storage');
+
+/* Controllers */
+
+var campaigninfoById = require('./controllers/campaigns').campaigninfoById;
+
+/* App variables */
 
 var app = express();
+var config = JSON.parse(fs.readFileSync('config.json'));
```

Comment:
Personally, I don't think it's an issue right now...we can re-visit when our configuration grows big.

I had thought `config/config.json` was the agreed convention between @kgdiem and yourself and an oversight here. :)

That being said, I'm onboard with this convention.